config/tsconfig.typecheck.json

```diff
@@ -0,0 +1,34 @@
+{
+    "extends": "../tsconfig.json",
+    "include": [
+        "../packages/**/*.*"
+    ],
+    "compilerOptions": {
+        "emitDeclarationOnly": false,
+    },
+    "exclude": [
+        "../packages/**/dist/**/*.*",
+        // some packages have internal reference to file so cannot exclude them indivdually 
```

Comment:
as far as I know comments like that are not allowed in json files

Reply:
i think due to the lack of the eslint rule it did not show up , will make sure next time